src/OpenTelemetry.Exporter.ZPages/Implementation/ZPagesActivities.cs

```diff
@@ -0,0 +1,126 @@
+﻿// <copyright file="ZPagesActivities.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Timers;
+#if NET452
+using OpenTelemetry.Internal;
+#endif
+
+namespace OpenTelemetry.Exporter.ZPages.Implementation
+{
+    /// <summary>
+    /// Implements the ZPages Activity Queue which stores all the required activity information.
+    /// </summary>
+    public static class ZPagesActivities
```

Comment:
Should this be `internal`? Going off of the "Implementation" namespace. Also, something about the name seems off. Not a big deal if it's internal. Not sure what would be better, `ZPagesActivityQueue` or `ZPagesActivityTracker`? Those aren't great either.

Reply:
i think tracker gets a better naming. queue it assumes that we have queues, but we have dictionaries. i'm updating!